ext/web/lib.deno_web.d.ts

```diff
@@ -140,11 +140,22 @@ declare type EventListenerOrEventListenerObject =
   | EventListenerObject;
 
 interface AddEventListenerOptions extends EventListenerOptions {
+  /** A boolean value indicating that the listener should be invoked at most
```

Comment:
Not sure if this has been the way for Deno doc comments, but personally I've used backticks to separate "code references" from normal doc text. eg. To align with the MDN docs, I'd use  \`listener\` to have it show up like this: `listener`.